Update library

addyi opened the pull request:
- Disable jetifier
- Update dependencies
- Update library version to 1.2.0

Commit:
Disable jetifier

Commit:
Update dependencies

Commit:
Update targetSdkVersion

Commit:
Update library version to 1.2.0

androideveloper commented on library/build.gradle:
```diff
@@ -3,35 +3,31 @@ apply plugin: 'kotlin-android'
+        minSdkVersion 15
+        targetSdkVersion 30
+        versionCode 1
+        versionName "1.2.0"
```
You probably also want to update readme file with 
`implementation 'com.github.teresaholfeld:Stories:1.2.0'`

addyi replied:
done with c3bd9fc

androideveloper commented on app/build.gradle:
```diff
@@ -2,34 +2,29 @@ apply plugin: 'com.android.application'
-      minifyEnabled false
-      proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+    compileSdkVersion 30
+    buildToolsVersion '28.0.3'
```
Please remove this line

> The specified Android SDK Build Tools version (28.0.3) is ignored, as it is below the minimum supported version (29.0.2) for Android Gradle Plugin 4.1.3.

addyi replied:
done with 4f848a5

androideveloper commented on library/build.gradle:
```diff
@@ -3,35 +3,31 @@ apply plugin: 'kotlin-android'
-  buildToolsVersion '28.0.3'
-  group = "com.teresaholfeld.stories"
+    compileSdkVersion 30
+    buildToolsVersion '28.0.3'
```
Please remove this line

> The specified Android SDK Build Tools version (28.0.3) is ignored, as it is below the minimum supported version (29.0.2) for Android Gradle Plugin 4.1.3.

addyi replied:
done with 4f848a5

androideveloper commented on gradle/wrapper/gradle-wrapper.properties:
```diff
@@ -1,6 +1,6 @@
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```
bin -> all

Diff https://stackoverflow.com/a/25456329/1635488

addyi replied:
done with 73662f1

androideveloper reviewed (changes requested):
Done! Added a couple of comments below

Commit:
Update readme

Commit:
Remove unsupported build tool definition

The specified Android SDK Build Tools version (28.0.3) is ignored,
as it is below the minimum supported version (29.0.2) for Android Gradle Plugin 4.1.3.

Commit:
Change gradle wrapper from bin to all

omatuzenko-stocard commented on build.gradle:
```diff
@@ -1,14 +1,14 @@
+  ext.kotlin_version = '1.4.32'
   repositories {
     google()
     jcenter()
```
Is it possible to remove jcenter here?

androideveloper replied:
There is one more below, would be great to also remove it
I checked, seems that `com.github.dcendents:android-maven-gradle-plugin` is available on mavencentral. But it's deprecated project anyway, so maybe we can remove it?

omatuzenko-stocard reviewed (approved):
Just one remark about jcenter and travis build check fail if there is something we need to check

androideveloper reviewed (approved):
Looks good! Let's see if they accept the changes ^^

Commit:
Remove travis

Commit:
Remove jcenter
